varia/sass/layout/_structure.scss

```
@@ -87,6 +87,7 @@
 .site-main > article > *, // apply vertical margins to article level
 .site-main > .not-found > *, // apply vertical margins to article level
 .entry-content > *,
+.entry-content p, //ensure proper margins in Premium Content block
```

Do you think we could make this change more specific to the Premium content in particular? since the theme seems to be wanting to target only the direct siblings of `.entry-content` maybe our fix could live in the `sass/blocks/`

---

I think this approach is good. It is quite heavy handed, but I do think that all paragraph blocks should behave like this even if they are in other blocks. I think a second opinion from @kjellr would be useful though.

---

I think the intuition to want to target the premium content block is good, but in this case I agree with @scruffian that all paragraph blocks would benefit from applying the margins this way. 

Also worth noting that doing so means that any paragraph which is the first/last child child of its container will have a `margin-top/bottom: 0;` applied.

https://github.com/Automattic/themes/blob/6a6fbeb8111e08a8f2a4c89ca8daf81d1e6c8b13/varia/sass/layout/_structure.scss#L102

The comment should probably be updated though because it's not specific to the Premium Content block.
```suggestion
.entry-content p, // ensure all paragraphs, including nested ones, carry a margin
```